### PR TITLE
Feature: Partial Team Reg

### PR DESCRIPTION
**Pull request by Joe2k**

### CODE REVIEW

**Review by SibiAkkash — COMMENTED** [review]
lgtm, we could refactor in some places, i see a bunch of ternaries in the `create_transaction` and in the `RazorpayPayment` component that is slightly hard to read. And the create transaction function is also quite big, could think about breaking it up, could make it easier to read. 
Otherwise all good

File: frontend/src/components/TeamRegistration.js [hunk 1]
```diff
@@ -349,6 +396,69 @@ const TeamRegistration = () => {
...
+                    <div class="flex items-center gap-x-4">
+                      <img
+                        src={team.image ?? team.image_url}
+                        class="h-8 w-8 rounded-full"
+                      />
+                      <span class="font-medium">
+                        {team.name}{" "}
+                        <span class="me-2 rounded bg-red-100 px-2.5 py-0.5 text-xs font-medium text-red-800 dark:bg-red-900 dark:text-red-300">
+                          Partial
+                        </span>
```
Comment (SibiAkkash) [inline frontend/src/components/TeamRegistration.js:412]: Should we should this only to the team admin ? I don't think all users need to know the teams' payment status right ?

**Review by SibiAkkash — CHANGES_REQUESTED** [review]

File: frontend/src/components/TeamRegistration.js (hunk 1 above)
Comment (Joe2k) [reply frontend/src/components/TeamRegistration.js:412]: Hmm I thought it be better cause OPs also would wanna know and it avoids misconception

Comment (SibiAkkash) [reply frontend/src/components/TeamRegistration.js:412]: right, makes sense. We can think about an ops role for the Hub later on maybe, for things like this, and if there's a need. Fine for now

**Pull request merged**